setup/updating.md

```diff
@@ -44,13 +44,15 @@ To update Coder, follow these steps:
     helm get values --namespace coder coder > current-values.yml
     ```
 
+    > Double-check your values to ensure it only contains your changes.  
+
```

Comment:
Context: My values file contained references of old 1.16 coder pods when I was attempting to upgrade to 1.17

```sh
➜  cdr-cs git:(main) ✗ helm get values --namespace coder coder > current-values.yml
➜  cdr-cs git:(main) ✗ cat current-values.yml 
USER-SUPPLIED VALUES:
USER-SUPPLIED VALUES: null
cemanager:
  accessURL: ""
  image: docker.io/coderenvs/coder-service:1.16.1
...
```